docs/modules/player-lifecycle/basic-player-creation.md

```diff
@@ -0,0 +1,94 @@
+<%(TOC)%>
+# Implementing basic player creation
+
+<%(Callout message="
+Before reading this document, make sure you have read the following documentation on:
+
+* [Player Lifecycle Feature Module]({{urlRoot}}/modules/player-lifecycle/overview)
+* [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+* [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+By default, the module automatically sends a player creation request as soon as the client-worker instance connects to SpatialOS. This spawns a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) representing a player.
+
+To do this, you need to:
+
+1. Set up your worker connector.
+1. Define the entity template for your player entities.
+1. Configure what entity template to use for player creation.
```

Comment:
I'm not a huge fan of this preamble for a short number of steps. I defer to @jessicafalk or @ElleEss on what our policy is for this. 

Also saying `To do this` after saying it automatically spawns a player feels weird

Reply:
usually I do like lists to emphasize of what's gonna happen next, but in this case I agree with Jamie.  Can get rid of this, the steps are really short :)